app/Service/StudentDetailsService.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Service;
+
+use App\Exceptions\StudentNotFoundException;
+use App\Models\Resume;
+use App\Models\Student;
+use App\Models\Tag;
+
+class StudentDetailsService
+{
+    public function execute($student)
```

Comment:
According with the comment made in the controller, I believe using ``$studentId`` instead of ``$student`` makes the code more readable.

Reply:
Done